components/notifications-service/server/lib/data/migrations.ex

```diff
@@ -55,7 +55,8 @@ defmodule Notifications.Data.Migrator.Migrations do
      %{description: "Add critical_controls_only to rules table",
        queries: [
         "ALTER TABLE rules ADD COLUMN critical_controls_only BOOLEAN;",
-        "ALTER TABLE rules ALTER COLUMN critical_controls_only SET DEFAULT FALSE;"]
+        "ALTER TABLE rules ALTER COLUMN critical_controls_only SET DEFAULT FALSE;",
+        "UPDATE rules SET critical_controls_only=FALSE;"]
```

Comment:
I know too little about this service's data migration setup -- but in other services, touching a migration that has already been deployed (to dev, or acceptance) would _not_ lead to that code being run again. Is that **not** the case here? (In other words, to fix this situation, I'd have expected _another migration_ that's run after this one... hope that makes sense...)

Reply:
done